Projects/AccountSDKIOSWebExample/ExampleWeb/ContentView.swift

```diff
@@ -66,6 +68,33 @@ struct ContentView: View {
         }.sheet(isPresented: $isPresentedSimplifiedLogin, content: { SimplifiedLoginSwiftUIView() })
     }
     
+    func storeToSharedKeychain() {
+        let myAccessGroup = "GS8T83EM2X.com.testing.SharingExample"
+        let manager = SimplifiedLoginManager(accessGroup: myAccessGroup, client: ExampleWebApp.client)
+        manager.storeInSharedKeychain(clientId: "AppOne", aStringValue: "THIS IS APP ONE STORING :D") { result in
+            switch result {
+            case .success:
+                print("Stored to SHARED KEYCHAIN. BOOYA!")
+            case .failure(let error):
+                print("FAILED TO STORE, error: \(error.localizedDescription)")
+            }
+        }
+    }
+    
+    func fetchFromSharedKeychain() {
+        let myAccessGroup = "GS8T83EM2X.com.testing.SharingExample"
+        let manager = SimplifiedLoginManager(accessGroup: myAccessGroup, client: ExampleWebApp.client)
```

Comment:
can we persist SimplifiedLoginManager for both these methods? Also extract to class/struct accessGroup string?